src/olympia/amo/models.py

```diff
@@ -399,6 +399,19 @@ class Meta:
     def get_absolute_url(self, *args, **kwargs):
         return self.get_url_path(*args, **kwargs)
 
+
+class ModelBase(caching.base.CachingMixin, UncachedModelBase, models.Model):
+    """
+    Base class for AMO models to abstract some common features.
+
+    * Adds automatic created and modified fields to the model.
+    * Fetches all translations in one subsequent query during initialization.
```

Comment:
I'm not sure if it matters but both of these comments seem to apply not to this class but to the parent classes now.

Reply:
They both apply to both base classes and that's why I chose to copy/paste them (avoids digging through classes when you are debugging)